tests/admin_views/admin.py

```diff
@@ -767,8 +767,23 @@ class RelatedPrepopulatedInline3(admin.TabularInline):
     autocomplete_fields = ['fk', 'm2m']
 
 
+class RelatedPrepopulatedInline4(admin.StackedInline):
+    model = RelatedPrepopulated
+    extra = 1
+    fields = ['name', 'pubdate', 'status', 'slug1', 'slug2']
+    prepopulated_fields = {
+        'slug1': ['name', 'pubdate'],
+        'slug2': ['status', 'name'],
+    }
```

Comment:
We don't need two slugs for a regression test. I would leave only one, e.g.
```suggestion
    fields = ['name', 'pubdate', 'slug1']
    prepopulated_fields = {'slug1': ['name', 'pubdate']}
```

Reply:
Actually, `slug1` and `slug2` are required fields in `RelatedPrepopulated` model. I think the test won't work if we do not populate both the slugs.

Reply:
OK, we can at least simplify `slug2`, e.g.:
```python
prepopulated_fields = {
   'slug1': ['name', 'pubdate'],
   'slug2': ['status'],
}
```

Reply:
Okay, will do that.